### ISSUE TITLE
Anyone who wants ownership/permissions, please request here

### ISSUE THREAD

**Issue by GregRos** [GregRos/IEMod.pw#91]
I'm the person who opened this repository and did a lot of IE Mod development a while ago. I'm not really interested in the game anymore, so I probably won't be developing the mod further. I may continue development of Patchwork though.

The repository is owned by the "IEMod" organization. Anyone with ownership of the organization also has complete control over the repository and can add new members/owners. 

If you want the ability to push to this repository or total ownership, please make a request here. Me or someone else will probably get around to answering it.

**Comment by Alakabon** [comment]
Hey ! 

I've got some time this summer to update it to the latest patches and try to fix some broken components. I've worked on it a bit in the past for the NPC stats and weapon glow modules.

I'd like the ability to modify things on the nexus mostly, there is a whole lot of stuff there that is a bit confusing.

If you'd prefer to keep it intact, would it be okay for me to make a new page for the mod with only the latest stuff on it? I'll credit the whole team obviously.

**Comment by GregRos** [comment]
Sure, whatever you like. Just give me your nexus details.

**Comment by BluesKutya** [comment]
Hi,

I also updated some file for the latest game version, and I like to update the mod in nexus if you don't mind.

**Comment by GregRos** [comment]
@BluesKutya Give me your nexus details. Do you want to get ownership for the repo?

I'm still working on Patchwork sometimes (Rarely) but not on IEMod anymore.

**Comment by BluesKutya** [comment]
Hi GregRos,
At first thank you for your hard work and thanks for the fast reply.

Alakabon have given me access to IEMod nexus mod.
So today I uploaded two versions for missing game versions.

I would like to merge back (with pull request) the update commits.
Or do you can give me permission for IEMod repo?

**Comment by GregRos** [comment]
@BluesKutya You should have an invite to join the IEMod organization. You can merge it yourself :smile:

**Comment by BluesKutya** [comment]
@GregRos thank you!
I'm on it

**Comment by grayshirk** [comment]
Hey yo, epic games release pretty much made the mod toast. I have no idea how any of this works and spent all day learning how to de-compile and clean c# syntax errors. Only to find there were none. _(I had no Visual studio, NET.Reflector, or unity knowledge before today. Obviously im still a novice but I can now navigate these programs lol)_ 

The error given apon testing is: 
**"An Error has occurred,
While trying to: Patch the game
Error Type: A system error or some sort of bug.
(Argument Null Exception)
Internal Message: Value cannot be null.
Parameter name: Key"**

The corresponding log is attached.
[log.txt](https://github.com/IEMod/IEMod.pw/files/5962526/log.txt)


In short, I learned stuff today and now want to learn Coding/Scripting.

Also Sorry for the Necropost
